### ISSUE TITLE
RANDOMSEARCH as reference algorithm for easy targets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [numbbo/coco#21]
```
Try RANDOMSEARCH as alternative reference algorithm for easy target values. 
Using a portfolio of the best of [-4,4], [-4.5, 4.5], [-5, 5] might be the 
right thing to do. However, this is not too realistic for dimension>10. 
Alternatively, we can sample cubes around the optimum and like that estimate 
the volume for each target level. The reference would be rel_vol with 
rel_vol^dimension<target. 
```

Original issue reported on code.google.com by `dimo.bro...@inria.fr` on 13 Jun 2014 at 4:43

**Comment by brockho** [comment]
The first suggestion from Aug 8, 2015 above should be relatively easy to test - now that any data set can be used as reference algorithm. However, when actually implementing a different reference algorithm for the standard and the runlength-based targets setting (aka "expensive"), issue #1198 needs to be considered.

**Comment by nikohansen** [comment]
I fully agree. It's very good that we can compare with random search and analysing level-set volumes of the testbeds remains IMHO an interesting approach, not necessarily an issue related to the COCO software though.

**Comment by nikohansen** [comment]
I found this

Average running time (aRT in number of f-evaluations as log10 value) divided by dimension versus dimension. The target function value is chosen such that the reference algorithm just failed to achieve an aRT of !!PPFIGS−FTARGET!!×DIM. 

in [this](http://coco.gforge.inria.fr/github/issue00021/ppdata-expensive-refRSportfolio/RANDO_BIPOP_BFGS__SMAC-_lmm-C_HCMA/ppfigs.html) caption.

**Comment by nikohansen** [comment]
In the scatter plot html legend I see now

<img width="657" alt="screen shot 2017-08-25 at 12 42 10" src="https://user-images.githubusercontent.com/7316439/29711455-27bbaccc-8995-11e7-9cb8-744ea7a1e869.png">

**Comment by brockho** [comment]
I have observed this as well and will take care of it.

**Comment by brockho** [comment]
The above mentioned problems should now be all solved with the latest pull request (#1505). However, my browser (Firefox under Windows) had problems to display the new markers in the caption of the scaling plots (the "varhexagon..." issue) and I hope that on other platforms, the problem does not occur. If so, we shall reopen this issue once again.
